### PR TITLE
Create location-awareness.md

### PR DESCRIPTION
**Pull request by Wenting0905**
PTAL @siddontang @shenli

### CODE REVIEW

File: op-guide/location-awareness.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+tikv-server --labels zone=z4,rack=r1,host=h1
+tikv-server --labels zone=z4,rack=r1,host=h2
+tikv-server --labels zone=z4,rack=r2,host=h1
+tikv-server --labels zone=z4,rack=r2,host=h2
+```
+
+In other words, 16 TiKV instances are distributed across 4 data zones, 8 racks and 16 machines.
+
+In this case, PD will schedule different replicas of each datum to different data zones.
+If one of the data zones hangs up, everything still works well.
```
Comment (siddontang) [inline op-guide/location-awareness.md]: use list for the three if

File: op-guide/location-awareness.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+If the data zone couldn't recover within a period of time, PD will remove the replica of this data zone.
+If there are only three (or two) data zones, we cannot guarantee that everything would work well when a data zone hangs up. However, it is ensured that everything is fine when a site hangs up.
+
+In a word, PD maximized the disaster recovery of the cluster according to the current topology. Therefore, if we want to reach to a certain level of disaster recovery, we need to deploy many machines in different sites according to the topology. The number of machines should be more than the one of `max-replicas`.
```
Comment (siddontang) [inline op-guide/location-awareness.md]: /maximized/maximizes/s

**Comment by siddontang** [comment]
Rest LGTM

File: op-guide/location-awareness.md (hunk 2 above)
Comment (Wenting0905) [inline op-guide/location-awareness.md]: you mean change to present tense or use more tenses? and what does the "s" stand for?

Comment (Wenting0905) [inline op-guide/location-awareness.md]: you mean change to present tense or add more tenses? and what does the "s" stand for?

**Comment by Wenting0905** [comment]
@shenli PTAL and please refer to https://github.com/pingcap/docs-cn/blob/master/op-guide/location-awareness.md to find the Chinese version.

**Comment by queenypingcap** [comment]
@shenli PTAL.

File: op-guide/location-awareness.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+## Overview
+
+PD schedules according to the topology of the TiKV cluster to maximize the TiKV's capability for disaster recovery.
+
+Before you begin, see [Binary Deployment (Recommended)](./binary-deployment.md) and [Docker Deployment](./docker-deployment.md).
+
+## TiKV reports the topological information
+
+TiKV reports the topological information to PD through the startup parameter or configuration of TiKV.
```
Comment (shenli) [inline op-guide/location-awareness.md]: through -> according to

**Comment by queenypingcap** [comment]
@shenli @siddontang  PTAL

**Comment by shenli** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**